src/components/icon-box/utils.ts

```diff
@@ -34,7 +37,10 @@ export function getTransportModeSvg(mode?: AnyMode, subMode?: AnySubMode) {
       if (APP_ORG === 'atb' && subMode === 'shuttleBus') {
         return {svg: SvgShuttle, name: 'Shuttle'};
       }
-      return {svg: Bus, name: 'Bus'};
+
+      return subMode && TRANSPORT_SUB_MODES_BUS.includes(subMode)
```

Comment:
Don't think we need the TRANSPORT_SUB_MODES_BUS array here when it's only the one mode. A `if (submode === TransportSubmode.NightBus)` check should be good enough.

Also noticed I should have used `TransportSubmode.ShuttleBus` in the check above instead of `'shuttleBus'`, so feel free to update that if you want 😄